clients/drcachesim/scheduler/scheduler.h

```diff
@@ -1331,6 +1331,39 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
         uint64_t wait_start_time = 0;
     };
 
+    // Used for reading as-traced schedules.
+    struct schedule_output_tracker_t {
+        schedule_output_tracker_t(bool valid, input_ordinal_t input,
+                                  uint64_t start_instruction, uint64_t timestamp)
+            : valid(valid)
+            , input(input)
+            , start_instruction(start_instruction)
+            , stop_instruction(0)
+            , timestamp(timestamp)
+        {
+        }
+        bool valid;
```

Comment:
Document when an entry wouldn't be "valid".

Reply:
Done.